etc/config/c++.amazon.properties

```diff
@@ -761,6 +761,34 @@ compiler.clang-rocm-60102.semver=rocm-6.1.2
 compiler.clang-rocm-60102.options=--gcc-toolchain=/opt/compiler-explorer/gcc-12.2.0
 compiler.clang-rocm-60102.ldPath=${exePath}/../lib|${exePath}/../lib/x86_64-unknown-linux-gnu
 
+group.swift-clang.compilers=swift-nightly-clang:swift-devsnapshot-clang:swift603-clang
+group.swift-clang.intelAsm=-mllvm --x86-asm-syntax=intel
+group.swift-clang.options=-fuse-ld=lld
```

Comment:
Is there a better way to force lld as the linker? swift's clang driver on linux defaults to GNU gold, but that compiler is unmaintained and deprecated.  <https://github.com/swiftlang/swift/issues/79163>

Reply:
What do you mean by better? This is the clang flag to choose the linker, what else do you need?